tests/utils/test_text_util.py

```diff
@@ -159,6 +159,33 @@ def test_hash_list(self):
         self.assertEqual(result_from_list, hash_from_string.digest())
         self.assertEqual(hash_from_string.hexdigest(), '6367c48dd193d56ea7b0baad25b19455e529f5ee')
 
+    def test_empty_strings(self):
+        self.assertTrue(textutil.is_str_none_or_whitespace(' '))
+        self.assertTrue(textutil.is_str_none_or_whitespace('\t'))
+        self.assertTrue(textutil.is_str_none_or_whitespace('\n'))
+        self.assertTrue(textutil.is_str_none_or_whitespace(' \t'))
+        self.assertTrue(textutil.is_str_none_or_whitespace(' \r\n'))
+
+        self.assertTrue(textutil.is_str_empty(' '))
+        self.assertTrue(textutil.is_str_empty('\t'))
+        self.assertTrue(textutil.is_str_empty('\n'))
+        self.assertTrue(textutil.is_str_empty(' \t'))
+        self.assertTrue(textutil.is_str_empty(' \r\n'))
+
+        self.assertFalse(textutil.is_str_none_or_whitespace(u' \x01 '))
+        self.assertFalse(textutil.is_str_none_or_whitespace(u'foo'))
+        self.assertFalse(textutil.is_str_none_or_whitespace('bar'))
+
+        self.assertFalse(textutil.is_str_empty(u' \x01 '))
+        self.assertFalse(textutil.is_str_empty(u'foo'))
+        self.assertFalse(textutil.is_str_empty('bar'))
+
+        self.assertFalse(textutil.is_str_none_or_whitespace(u'\x00'))
+        self.assertFalse(textutil.is_str_none_or_whitespace(u' \x00 '))
+
+        self.assertTrue(textutil.is_str_empty(u'\x00'))
+        self.assertTrue(textutil.is_str_empty(u' \x00 '))
```

Comment:
Consider pulling these strings into named variables (which can live within this method) so you can be certain you're invoking the two methods with identical strings.

Reply:
Updated